### PR TITLE
Have done extra task for lesson-14

### PR DESCRIPTION
**Pull request by targonskii**
Have done task with * mark.

### CODE REVIEW

File: lesson-14extra/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <link rel="stylesheet" href="./main.css">
+  <title>Lesson-14 Extra</title>
+</head>
+<body>
+  <form id='formEl' class="form">
+    <div class="addImg">
+      <p>Click to add your image</p>
+      <input type="file" name="file" hidden="true">
+    </div>
+    <script src="./main.js"></script>
```
Comment (nikitik-nikita) [inline lesson-14extra/index.html]: Script in wrong place

File: lesson-14extra/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./main.css">
+  <title>Lesson-14 Extra</title>
+</head>
+<body>
+  <form id='formEl' class="form">
```
Comment (nikitik-nikita) [inline lesson-14extra/index.html]: Notice the quotation marks. Let it all be in the same style

**Review by nikitik-nikita — COMMENTED** [review]
I think you should break down the logic of the `clickDiv` function. Each new click adds a picture, not replaces the old one

File: lesson-14extra/index.html (hunk 1 above)
Comment (targonskii) [reply lesson-14extra/index.html]: Fixed

File: lesson-14extra/index.html (hunk 2 above)
Comment (targonskii) [reply lesson-14extra/index.html]: Fixed

**Comment by targonskii** [comment]
> I think you should break down the logic of the `clickDiv` function. Each new click adds a picture, not replaces the old one

Changed logic

**Pull request merged**